contribs/gmf/options/gmfx.js

```diff
@@ -112,6 +112,143 @@ gmfx.GridSource.prototype.source;
+ * altitude: (number|undefined),
+ * color_packed: Array<(number|undefined)>,
+ * coords: Array<(number|undefined)>,
+ * intensity: (number|undefined),
+ * classification: (number|undefined),
+ * set: (boolean|undefined)
+ * },
+ * pEnd: {
+ * distance: (number|undefined),
+ * altitude: (number|undefined),
+ * color_packed: Array<(number|undefined)>,
+ * coords: Array<(number|undefined)>,
+ * intensity: (number|undefined),
+ * classification: (number|undefined),
+ * set: (boolean|undefined)
+ * }
+ * }}
+ */
+gmfx.lidarMeasure;
+
+/**
+ * The lidar point attribute: describes the point binary property, material, visibility, ...
+ * @typedef {{
+ *   bytes: (string|undefined),
+ *   elements: (number|undefined),
+ *   name: (string|undefined),
+ *   value: (string|undefined),
+ *   visible: (number|undefined)
+ * }}
+ */
+gmfx.lidarPointAttribute;
+
+/**
+ * The lidar point attribute list width default option
+ * @typedef {{
+ *   availableOptions: (Array.<gmfx.lidarPointAttribute>|undefined),
+ *   selectedOption: (gmfx.lidarPointAttribute|undefined)
+ * }}
+ */
+gmfx.lidarPointAttributeList;
+
+/**
+* @typedef {{
+*  color: (string|undefined),
+*  name: (string|undefined),
+*  value: (string|undefined),
+*  visible: (boolean|undefined)
+*}}
+*/
+gmfx.lidarClassification;
+
+/**
+* @typedef {{
+*   number: gmfx.lidarClassification
+}}
+*/
+gmfx.lidarClassificationObjects;
+
+/**
+ * The lidar point classification: describes the point binary property, material, visibility, ...
+ * @typedef {{
+ *   color: (Array.<number>|undefined),
+ *   name: (string|undefined),
+ *   value: (string|undefined),
+ *   visible: (number|undefined)
+ * }}
+ */
+gmfx.lidarPointClassification;
+
+
+// /**
+//  * The object containing all points in profile
+//  * @typedef {{
+//  * scaleX: (Object|undefined),
+//  * currentScaleY: (Object|undefined),
+//  * previousDomainX: (Array.<number>|undefined),
+//  * previousDomainY: (Array.<number>|undefined),
+//  * currentZoom: (number|undefined),
+//  * distanceOffset: (number|undefined),
+//  * margin: {
+//  *  left: (number|undefined),
+//  *  top: (number|undefined),
+//  *  right: (number|undefined),
+//  *  bottom: (number|undefined)
+//  * },
+//  * tolerance: (number|undefined),
+//  * configLoaded: (boolean),
+//  * pointAttributes: (gmfx.lidarPointAttributeList),
+//  * classification: Object,
+//  * profilWidth: (number|undefined),
+//  * autoWidth: (boolean|undefined),
+//  * minLOD: (number),
+//  * initialLOD: (number|undefined),
+//  * pointSize: (number|undefined),
+//  * maxLevels: (Object|undefined),
+//  * maxPoints: (number|undefined),
+//  * defaultColor: (string|undefined),
+//  * pointClouds: (Object|undefined),
+//  * pointAttributesRaw: (gmfx.lidarPointAttributeList),
+//  * defaultAttribute: (gmfx.lidarPointAttribute),
+//  * defaultPointAttribute: (string|undefined),
+//  * debug: (boolean|undefined),
+//  * pytreeLinestring: (string|undefined)
+//  * }}
+//  */
+// gmfx.LidarProfileConfig;
```

Comment:
To remove ?

Reply:
To be used for typing Pytree config service ouput. Help needed on this point...